package.json

```diff
@@ -13,6 +13,9 @@
     "cypress:gh": "synpress run --record --key 9d633f22-7630-4a11-9cf4-ff9809332564 --configFile cypress.config.ts --group Desktop -b chromium",
     "find-unused-modules": "ts-prune -s .graphql.ts -i pages/"
   },
+  "resolutions": {
+    "@types/react": "^18.0.0"
+  },
```

Comment:
Need to make sure any third party libs aren't bringing us down to the React 17 types.

Reply:
fantastic catch

Reply:
🧠